plugin/vimproc.vim

```diff
@@ -36,10 +36,26 @@ let s:save_cpo = &cpo
 set cpo&vim
 " }}}
 
+command! VimProcInstall call s:install()
 command! -nargs=+ -complete=shellcmd VimProcBang call s:bang(<q-args>)
 command! -nargs=+ -complete=shellcmd VimProcRead call s:read(<q-args>)
 
 " Command functions:
+function! s:install() "{{{
+  vert botright new " Create a new split window on the far right
+  setlocal buftype=nofile bufhidden=wipe nobuflisted noswapfile nowrap
+  lcd %:p:h:h " Change to the correct directory
+
+  if executable('make')
+    execute '$read !make'
+  else if executable('gmake')
+    execute '$read !gmake'
+  else if executable('nmake')
+    execute '$read !nmake'
+  endif
+
+  setlocal nomodifiable
```

Comment:
This is to preserve all output from `make`, to make it easier to debug a broken build.